Signals/Dto/MissingValuePolicy/SpecificValueMissingValuePolicy.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Dto.Conversions;
+
+namespace Dto.MissingValuePolicy
+{
+    public class SpecificValueMissingValuePolicy : MissingValuePolicy
+    {
+        public object Value { get; set; }
+
+        public Quality Quality { get; set; }
+
+        [MapFromGenericDataType]
```

Comment:
Dto is something visible to 'experiment subjects'. Just wondering if we should introduce them something like this...